CMakeLists.txt

```diff
@@ -109,6 +109,15 @@ if(NOT CMAKE_CONFIGURATION_TYPES)
 endif()
 
 option(QT6 "Build with Qt6" OFF)
+if(APPLE)
+option(QOPENGL "Use QOpenGL... classes instead of QGLWidget" ON)
+else()
+option(QOPENGL "Use QOpenGL... classes instead of QGLWidget" OFF)
+endif()
+
+if(QOPENGL)
+    add_compile_definitions(MIXXX_USE_QOPENGL)
+endif()
```

Comment:
Style nit, but we'd probably want to use two-space indent for both of these.

Reply:
got it